packages/edit-site/src/components/page-pages/index.js

```diff
@@ -48,17 +49,50 @@ export default function PagePages() {
 		hiddenFields: [ 'date', 'featured-image' ],
 		layout: {},
 	} );
-	// Request post statuses to get the proper labels.
+
 	const { records: statuses } = useEntityRecords( 'root', 'status' );
-	const postStatuses = useMemo(
-		() =>
-			statuses === null
-				? EMPTY_OBJECT
-				: Object.fromEntries(
-						statuses.map( ( { slug, name } ) => [ slug, name ] )
-				  ),
-		[ statuses ]
-	);
+	const { postStatuses, defaultStatuses } = useMemo( () => {
+		return {
+			postStatuses:
+				statuses === null
+					? EMPTY_OBJECT
+					: Object.fromEntries(
```

Comment:
`statuses` were handled initially like an `object` because with REST API, it was one. Now that it's an entity and we have an array, do we still need to all that? The difference would only be in `getValue` for status to `Array.find`.

Reply:
I don't have a strong opinion on doing it either way. Do you think we should update it in this PR?

Reply:
I think it's simplifies the code, but no strong opinions here.

Reply:
Done at https://github.com/WordPress/gutenberg/pull/55476/commits/1ca610cb7d5665ff082209e70e3a3397fe2c2540 Thanks for the nudge, it does indeed simplify the code.